styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  height: 100vh;
+  background-image: url('/images/image_geometry_1.svg');
+  background-repeat: no-repeat;
+}
+
+.hamburger {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.btn-icon {
+  background-image: url(/images/ic_menu.svg);
```

Comment:
- [ ] There is an issue with the hamburger menu icon, it is not being shown and the problem is because the `source path` is including the `/` before. As suggested before, you could include a `.` (dot) before in order to make it appear. It should look like: `background-image: url(./images/ic_menu.svg);` :hugs:

  ![image](https://user-images.githubusercontent.com/81522435/138361543-963c4c0e-e490-46a0-8a3f-dd702456c4e3.png)